README.md

```diff
@@ -210,106 +185,91 @@ Further, the key manager interface **must** be passed as an argument to _all_ pu
 > Consumers of our SDKs should be able to provide their own `KeyManager` implementations if desired
 <!-- markdownlint-disable-next-line -->
 
-> [!NOTE]
-> ⚠️ = implemented but no test vectors present
+### Decentralized Identifiers
 
-### [`did:web`](https://w3c-ccg.github.io/did-method-web/)
+#### [DID Documents](https://www.w3.org/TR/did-core/) & [DID Resolution](https://w3c-ccg.github.io/did-resolution/)
 
-| Feature      | Typescript | Kotlin | Rust | Swift |
-| ------------ | ---------- | ------ | ---- | ----- |
-| `Resolution` | ❌          | ❌      | ⚠️    | ❌     |
+Independent of DID Methods, we support DID Documents according to [DID Core v1.0](https://www.w3.org/TR/2022/REC-did-core-20220719/) with the adjustments specified in [this corresponding document](spec/did.md).
 
-### [`did:jwk`](https://github.com/quartzjer/did-jwk/blob/main/spec.md)
+#### DID Methods
 
-| Feature      | Typescript | Kotlin | Rust | Swift |
-| ------------ | ---------- | ------ | ---- | ----- |
-| `Creation`   | ❌          | ❌      | ⚠️    | ❌     |
-| `Resolution` | ❌          | ❌      | ⚠️    | ❌     |
+| Method                                                 | Creation   | Resolution | Note |
+| ------------------------------------------------------ | ---------- | ---------- | ---- |
+| [`did:web`](https://w3c-ccg.github.io/did-method-web/) | ❌         | ✅         | -   |
```

Comment:
What's our reasoning for not supporting `did:web` creation? Is it that hosting is assumed to be part of the creation function and we don't venture into the space of how to deploy & host the doc?

Reply:
There is a path forward where we can create a did:web similar to how we did it in SSI-Service, but it will be strange if you create a did:web and then try to resolve it right after it wont resolve (because you have not put it on your domain yet)

We should at least have a util function to create the did:web doc for people to put on their domain

Reply:
Fo sho, for reference that's what we have in web5-go [right now](https://github.com/TBD54566975/web5-go/blob/256e91138ecc83a77e99c59a8ba957f27cbe30ac/dids/didweb/didweb.go#L103)

Created [a ticket](https://github.com/TBD54566975/web5-go/issues/105) to track work there